### DISCUSSION TITLE
Welcome to LinuxGSM Discussions!

### DISCUSSION THREAD

**Discussion by dgibbs64** [GameServerManagers/LinuxGSM#3124] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cokeman8D** [comment]
Hi I'm very new to this but i've known github for years, yet i cannot believe this category is only 2 months aged!
Hails to everyone and keep it up!

**Comment by rosanyadav08** [comment]
Can we bind Linux drivers in windows like touchpad driver audio driver etc
